### DISCUSSION TITLE
Floating table header

### DISCUSSION THREAD

**Discussion by kernow** [pacemaker82/PredBat-Table-Card#29] [Ideas]
It would be great to have an option to turn on and off a floating table header. If off it would behave as it does now. If on the table header would always be at the top of the screen even when you scroll down the page.

Great work by the way!

**Comment by pacemaker82** [comment]
Thanks - yeah I had thought about that and tried it. Its difficult because the way cards are nested into the page. But will see if I can find a solution at some point.
